addons/l10n_it_xml_export/wizard/monthly_tax_report_xml_export.py

```diff
@@ -0,0 +1,144 @@
+        help="VAT number of the parent company.",
+        default=lambda x: x._get_default_parent_company_data()["vat_number"],
+    )
+    company_code = fields.Char(
+        string="Company Code",
+        help="Codice Fiscale of the company.",
+        default=lambda self: "".join([char for char in self.env.company.vat if char.isdigit()]) if self.env.company.vat else False,
+    )
+    intermediary_code = fields.Char(
+        string="Intermediary Code",
+        default=lambda self: self.env.company.account_representative_id.l10n_it_codice_fiscale if self.env.company.account_representative_id else False,
+    )
+    submission_commitment = fields.Selection(
+        string="Prepared by",
+        help="If a representative is involved, this field precises whether the declaration was done by the taxpayer or the representative.",
+        default="1",
+        selection=[
+            ("1", "the Taxpayer"),
+            ("2", "the Representative"),
+        ],
+    )
+    commitment_date = fields.Date(
+        string="Date of Commitment",
+        default=fields.Date.context_today,
+    )
+    subcontracting = fields.Boolean(string="Subcontracting", help="Check this if the company operates in sub-contracting.")
+    exceptional_events = fields.Boolean(string="Exceptional Events", help="Check this if the declaration is affected by exceptional events.")
+    extraordinary_operations = fields.Boolean(string="Extraordinary Operations", help="Check this if the company has undergone extraordinary operations.")
+    show_method = fields.Boolean(
+        string="Show Method",
+        help="Check this if you want to show the method field.",
+        compute="_compute_show_method",
+    )
+    method = fields.Selection(
+        string="Method",
+        help="Indicates which advance payment calculation method was used. The value of the advance payment can be found in the VP13 line of the report.",
+        selection=[
+            ("1", "1 - Historical Method"),
+            ("2", "2 - Forecasting Method"),
+            ("3", "3 - Analytical Method"),
+        ],
+    )
+
+    def _get_default_parent_company_data(self):
+        parent_id = self.env.company.parent_id or (hasattr(self.env.company, "company.account_tax_unit_ids") and self.env.company.account_tax_unit_ids[0].main_company_id)
+        return {
+            "parent_id": parent_id,
+            "vat_number": "".join([char for char in parent_id.vat if char.isdigit()]) if parent_id else False,
+        }
+
+    def _compute_show_method(self):
+        self.show_method = False
+
+        submissions_periodicity = hasattr(self.env.company, "account_tax_periodicity") and self.env.company.account_tax_periodicity
+        date_to = fields.Date.from_string(self.env.context['l10n_it_xml_export_monthly_tax_report_options']['date']['date_to'])
+        for wizard in self:
+            if submissions_periodicity == "trimester" and (date_to.month - 1) // 3 + 1 == 4:
```

Comment:
```suggestion
            if submissions_periodicity == "trimester" and (date_to.month - 1) // 3 == 3:
```

But that's not possible, is it?

Reply:
Why would it not be ? submitting in december -> (12 - 1) // 3 = 3

Reply:
I confused `//` and `%` :facepalm: